.coveragerc

```diff
@@ -1096,6 +1096,7 @@ omit =
     homeassistant/components/shelly/light.py
     homeassistant/components/shelly/number.py
     homeassistant/components/shelly/sensor.py
+    homeassistant/components/shelly/update.py
```

Comment:
We should add tests and not reduce coverage

Reply:
Well the update.py is a new file that did not yet exist, but I will look into tests.

Reply:
Alright I finished writing full tests for the update.py file